examples/cluster/index.ts

```diff
@@ -84,10 +84,10 @@ const coredns = new eks.Addon("coredns", {
 });
 
 // Export the clusters' kubeconfig.
-export const kubeconfig1 = cluster1.kubeconfig;
-export const kubeconfig2 = cluster2.kubeconfig;
-export const kubeconfig3 = cluster3.kubeconfig;
-export const kubeconfig4 = cluster4.kubeconfig;
+export const kubeconfig1: pulumi.Output<any> = cluster1.kubeconfig;
```

Comment:
This is necessary due to us linking the eks sdk for testing purposes. Without the type assertions, we'd run into `tsc` errors like:

```
index.ts:93:14 - error TS2742: The inferred type of 'kubeconfig1' cannot be named without a reference to '../../nodejs/eks/node_modules/@pulumi/pulumi'. This is likely not portable. A type annotation is necessary.
```